zipkin-collector/activemq/src/test/java/zipkin2/collector/activemq/ITActiveMQCollector.java

```diff
@@ -183,12 +192,12 @@ void messageWithMultipleSpans(SpanBytesEncoder encoder) throws Exception {
       }
     };
 
-    activemq.pushMessage(collector.queue, PROTO3.encodeList(spans));
-    activemq.pushMessage(collector.queue, PROTO3.encodeList(spans)); // tossed on error
-    activemq.pushMessage(collector.queue, PROTO3.encodeList(spans));
-
     collector = builder().storage(buildStorage(consumer)).build().start();
 
+    pushMessage(collector.queue, PROTO3.encodeList(spans));
```

Comment:
moved this as I needed a non stopped collector instance. It doesn't change the nature of the test